includes/Modules/Search_Console.php

```diff
@@ -460,6 +456,15 @@ protected function setup_info() {
 		);
 	}
 
+	/**
+	 * Get the configured Webmasters service instance.
+	 *
+	 * @return Google_Service|\Google_Service_Webmasters
+	 */
+	private function get_webmasters_service() {
```

Comment:
Same concern like on #492, I'd prefer to drop this method and use a class constant for the service (e.g. `SERVICE_WEBMASTERS`).

Reply:
@aaemnnosttv Can you please update this similar to my comment in https://github.com/google/site-kit-wp/pull/492#discussion_r321742874?

Reply:
Done!